lib/ddtrace/contrib/sequel/utils.rb

```diff
@@ -0,0 +1,43 @@
+require 'ddtrace/contrib/rails/utils'
+
+module Datadog
+  module Contrib
+    module Sequel
+      # TODO[manu]: write docs
+      module Utils
+        def adapter_name
+          @adapter_name ||= Datadog::Contrib::Rails::Utils.normalize_vendor(
+            db.adapter_scheme.to_s
+          )
+        end
+
+        def sanitize_sql(sql)
+          regexp = Regexp.new('(\'[\s\S][^\']*\'|\d*\.\d+|\d+|NULL)', Regexp::IGNORECASE)
```

Comment:
This regexp should be updated to this
```
REGEXP ||= Regexp.new(
            Regexp.union([
              /'(?:[^']|'')*?(?:\\'.*|'(?!'))/,
              /"(?:[^"]|"")*?(?:\\".*|"(?!"))/,
              /\b-?(?:[0-9]+\.)?[0-9]+([eE][+-]?[0-9]+)?\b/,
              /\b(?:true|false|null)\b/i,
              /0x[0-9a-fA-F]+/,
              /(?:#|--).*?(?=\r|\n|$)/i,
              /\/\*(?:[^\/]|\/[^*])*?(?:\*\/|\/\*.*)/
            ])
          )
```
Replace this following line to `sql.to_s.gsub(REGEXP, '?')`. This should help speed it up some as well as remove extraneous `?`.